ext/nnstreamer/tensor_source/tensor_src_tizensensor.c

```diff
@@ -582,7 +589,8 @@ _ts_configure_handle (GstTensorSrcTIZENSENSOR * self)
       GST_ERROR_OBJECT (self,
           "The requested sensor sequence %d for sensor %d is not available. The max-sequence is used instead",
           self->sequence, self->type);
-      self->sequence = count - 1;
+      self->sequence = 0;
```

Comment:
Question: what's the meaning of `self->sequence = 0`? Is it `max-sequence` mentioned in the above error message?

Reply:
sequence=0 means the first sensor device of the given sensor types.